tests/Unit/Http/Validation/ValidatorTest.php

```diff
@@ -60,6 +60,27 @@ public function testValidateChaining()
         ));
     }
 
+    /**
+     * @covers \Engelsystem\Http\Validation\Validator::validate
+     */
+    public function testValidateMultipleParameters()
```

Comment:
There was not yet a test for multiple parameters in a validation. Since I was now using one in the SettingsController, I added the test here, too.